<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>지출내역 자동화</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>271</width>
     <height>387</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>작성년월</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="targetYearMonth">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy. MM</string>
        </property>
        <property name="currentSectionIndex">
         <number>1</number>
        </property>
        <property name="date">
         <date>
          <year>2025</year>
          <month>3</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="title">
       <string>메일추출</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="verticalGroupBox">
         <property name="title">
          <string>Naver Mail</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>아이디</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="naverId">
              <property name="text">
               <string>yijs79@naver.com</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>패스워드</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="naverPassword"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="verticalGroupBox">
            <property name="title">
             <string>결제수단</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QCheckBox" name="checkKbCard">
                 <property name="text">
                  <string>KB</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkHyundaiCard">
                 <property name="text">
                  <string>현대</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QCheckBox" name="checkHanaCard">
                 <property name="text">
                  <string>하나</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkNaverPay">
                 <property name="text">
                  <string>네이버페이</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QCheckBox" name="checkKbank">
                 <property name="text">
                  <string>케이뱅크</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="extractSpentMail">
           <property name="text">
            <string>작업실행</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="maiExtractStatus">
           <property name="text">
            <string>작업전</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
